packages/plugins/qr-code/src/qr-code/index.ts

```diff
@@ -0,0 +1,6 @@
+import QRCode from "easyqrcodejs-nodejs";
+
+export const useQrCode = async (options) => {
+    const qrcode = new QRCode(options);
+    return await qrcode.toDataURL();
+};
```

Comment:
```suggestion
export const useQrCode = async (options: type???) => {
    return new QRCode(options);
};
```
Let's add some TS types, and let's return all the object, then project can choose how to use it.